.github/workflows/ci.yml

```diff
@@ -16,10 +16,10 @@ env:
   CARGO_TERM_COLOR: always
   DASEL_VERSION: https://github.com/TomWright/dasel/releases/download/v1.24.3/dasel_linux_amd64
   RUSTFLAGS: "-D warnings"
-  FUEL_CORE_VERSION: 0.19.1
+  FUEL_CORE_VERSION: 0.20.1
   RUST_VERSION: 1.70.0
   FORC_VERSION: 0.40.1
-  FORC_PATCH_BRANCH: ""
+  FORC_PATCH_BRANCH: "upgrade/fuel-core-v0.20.1"
```

Comment:
Let's create an issue to bring this back to `""` once `forc` is released 😁

Reply:
Made the issue [here](https://github.com/FuelLabs/fuels-rs/issues/1054).